CHANGES.md

```diff
@@ -3,12 +3,21 @@ Change Log
 
 ### 1.11 - 2015-07-01
 
+* Deprecated
+  * Deprecated `AxisAlignedBoundingBox.intersect` and `BoundingSphere.intersect`.  These will be removed in 1.13.  Use `.intersectPlane` and, if necessary, `Plane.fromCartesian4`.
+  * Deprecated the `ObjectOrientedBoundingBox` class.  It will be removed in 1.12.  Use `OrientedBoundingBox` instead.
 * Improved the algorithm that `Camera.viewRectangle` uses to select the position of the camera, so that the specified rectangle is now better centered on the screen [#2764](https://github.com/AnalyticalGraphicsInc/cesium/issues/2764).
 * The performance statistics displayed by setting `scene.debugShowFramesPerSecond` to `true` can now be styled using the `cesium-performanceDisplay` CSS classes in `shared.css` [#2779](https://github.com/AnalyticalGraphicsInc/cesium/issues/2779).
 * Fixed a crash when `viewer.zoomTo` or `viewer.flyTo` were called immediately before or during a scene morph [#2775](https://github.com/AnalyticalGraphicsInc/cesium/issues/2775).
 * Fixed an issue where `Camera` functions would throw an exception if used from within a `Scene.morphComplete` callback [#2776](https://github.com/AnalyticalGraphicsInc/cesium/issues/2776).
 * `Model` can now load Binary glTF from a Uint8Array.
 * Added a new camera mode for horizon views. When the camera is looking at the horizon and a point on terrain above the camera is picked, the camera moves in the plane containing the camera position, up and right vectors.
+* Added `Matrix2`/`Matrix3`/`Matrix4.ZERO` constants for zero matrices.
+* Added `Matrix2`/`Matrix3.multiplyByScale` for multiplying against non-uniform scales.
+* Added `EllipsoidTangentPlane.projectPoint(s)ToNearestOnPlane` to project 3D points to the nearest 2D point on an `EllipsoidTangentPlane`.
+* Added `OrientedBoundingBox` which is an optimized/simplified bounding box for optimization use.
```

Comment:
Sounds awkward: "**optimized**/simplified bounding box for **optimization** use."

We could probably just say "Added `OrientedBoundingBox`" since 99% of users will not use this directly in the near future.